project/templates/project/components/dashboard/consommation.html

```diff
@@ -322,14 +322,38 @@ <h6 class="fr-mt-2w">Données</h6>
+                <div class="guide-content">
+                    <div class="guide-title">Comprendre les données</div>
+                    <div class="guide-description">
+                        <p class="fr-text--sm">
+                            Ce graphique représente chaque territoire par un rectangle, dont la taille est proportionnelle à sa surface.
+                            Plus le rectangle est grand, plus le territoire est vaste.
+                            La couleur des rectangles reflète le rapport entre la surface d’espaces NAF consommée et la surface totale du territoire.
+                            Plus la couleur est intense, plus ce rapport est élevé.
```

Comment:
Qu'est-ce que tu penses de cette dernière phrase à la place ?

```suggestion
                            Plus la couleur est foncée, plus la consommation d'espace est intense.
```